src/common/utils/document.ts

```diff
@@ -0,0 +1,5 @@
+import { WrappedDocument } from "@govtechsg/open-attestation";
+import { getData } from "@govtechsg/open-attestation";
+
+export const getDocumentId = (document: WrappedDocument) => `0x${document.signature.targetHash}`;
+export const getTokenRegistryAddress = (document: WrappedDocument) => getData(document).issuers[0].tokenRegistry;
```

Comment:
what about document store ? I know Tradetrust is not supposed to handle anymore (~) document with document store => however I' wondering what's the current behaviour with such file and also what do you expect from this function to return, string or string|undefined ?

Reply:
I dont want to move everything here yet, as I refactor I'll be moving things here if it is common

Reply:
doing just the necessary work without changing the whole repo